src/actions/locationsActions.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+import { setLocations, addsLocations } from './actions'
+
+/**
+ * Fetches locations 
+ * @param {*} dispatch 
+ */
+export function fetchLocations(dispatch) {
+  axios.get('http://localhost:3001/locations', {
+    headers: {
+      "Content-Type": "application/json",
+      "Authorization": "fake-access-token"
```

Comment:
you cannot hard code the access token, you need to use the one coming from the backend.